docs/en/core-services/Caching.md

```diff
@@ -204,70 +190,99 @@ A typical configuration will have three tiers for Memory, SSD, and HDD.
+    as the LRUAnnotator.
+    - The applicable configuration properties are `alluxio.worker.block.annotator.lrfu.step.factor` and
+    `alluxio.worker.block.annotator.lrfu.attenuation.factor`.
+
+The annotator utilized by workers is determined by the Alluxio property
+[`alluxio.worker.block.annotator.class`]({{ '/en/reference/Properties-List.html' | relativize_url }}#alluxio.worker.block.annotator.class).
 The property should specify the fully qualified class name within the configuration. Currently
 available options are:
 
-- `alluxio.worker.block.evictor.LRUEvictor`
-- `alluxio.worker.block.evictor.GreedyEvictor`
-- `alluxio.worker.block.evictor.LRFUEvictor`
-- `alluxio.worker.block.evictor.PartialLRUEvictor`
+- `alluxio.worker.block.annotator.LRUAnnotator`
+- `alluxio.worker.block.annotator.LRFUAnnotator`
+
+#### Evictor Emulation
+The old eviction policies are now deprecated and Alluxio provided implementations are replaced with appropriate annotation policies.
+Configuring old Alluxio evictors will cause worker startup failure with `java.lang.ClassNotFoundException`.
+
+However, Alluxio supports emulation mode which annotates blocks based on custom evictor implementation. The emulation assumes the configured eviction policy creates
+an eviction plan based on some kind of order and works by regularly extracting this order to be used in annotation activities.
+
+The old evictor configurations should be changes as below. (Failing to change the lef-over configuration will cause class load exceptions as old evictor implementations are deleted.)
+- LRUEvictor -> LRUAnnotator
+- GreedyEvictor -> LRUAnnotator
+- PartialLRUEvictor -> LRUAnnotator
+- LRFUEvictor -> LRFUAnnotator
+
+### Tiered Storage Management
+As block allocation/eviction no longer enforces a particular tier for new writes, a new block could end up in any configured tier.
+To enforce the assumption that tiers are configured from fastest to slowest, Alluxio now moves blocks around tiers based on block annotation policies.
+
+Below configuration is honoured by each individual tier management task:
+- `alluxio.worker.management.task.thread.count`: How many threads to use for management tasks. (Default:`CPU core count`)
+- `alluxio.worker.management.block.transfer.concurrency.limit`: How many block transfers can execute concurrently. (Default:`CPU core count`/ 2)
+
+#### Block Aligning (Dynamic Block Placement)
+Alluxio will dynamically move blocks across tiers in order to have block composition that is in line with the configured block annotation policy.
+
+To compensate this, Alluxio watches the I/O pattern and reorganize blocks across tiers to make sure
+`the lowest block of a higher tier has higher order than the highest block of a tier below`.
+
+This is achieved by `align` management task. This task, upon detecting tiers are out of order, swaps blocks among tiers
```

Comment:
By reading through this section, it looks to me these align or promote tasks are background thread pool. But it is not clear as users may wonder how these tasks affect the I/O performance of foreground tasks. Can you make this more explicit?

Reply:
added a link to "management task back-off" section which discusses this particular issue.